server/index.js

```diff
@@ -40,10 +42,12 @@ app.get('/getAllBooks', getAllBooks);
 app.get('/getOneBook/:id', getOneBook);
 app.get('/getPagesForBook/:id', getAllPages);
 app.post('/createBook', createBook);
+app.use('/createPageInBook/:id', createPage);
 app.post('/editBook/:id', editBook);
 
 app.use(adminsOnly); // user must be admin to access endpoints below
 app.get('/people', getPeople);
+app.post('/deletePage', deletePage);
```

Comment:
I think the id of the page should be in the url. Same for the edits/deletes below this line.